spring-cloud-dataflow-server/docker-compose-influxdb.yml

```diff
@@ -0,0 +1,66 @@
+version: '3'
+
+services:
+  kafka:
+    image: wurstmeister/kafka:2.11-0.11.0.3
+    expose:
+      - "9092"
+    environment:
+      - KAFKA_ADVERTISED_PORT=9092
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+      - KAFKA_ADVERTISED_HOST_NAME=kafka
+    depends_on:
+      - zookeeper
+  zookeeper:
+    image: wurstmeister/zookeeper
+    expose:
+      - "2181"
+  dataflow-server:
+    image: springcloud/spring-cloud-dataflow-server:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set!}
+    container_name: dataflow-server
+    ports:
+      - "9393:9393"
+    environment:
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.brokers=kafka:9092
+      - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.zkNodes=zookeeper:2181
+      - spring.cloud.skipper.client.serverUri=http://skipper-server:7577/api
```

Comment:
We need to include the fix made from https://github.com/spring-cloud/spring-cloud-dataflow/commit/f96aa1a307cf415c9b63fe298ff61ab0938745eb